examples/shallow_water/moist_thermal_williamson5.py

```diff
@@ -34,8 +34,8 @@
 mu2 = 0.98
 q0 = 135  # chosen to give an initial max vapour of approx 0.02
 beta2 = 10
-qprecip = 10e-4
-gamma_r = 10e-3
+qprecip = 1e-4
```

Comment:
can you comment on why these have changed?

Reply:
The original thresholds came from Zerroukat and Allen (2015) but in our model there is less cloud produced than in that paper. Using the threshold for rain from Zerroukat and Allen (2015) would mean that our model would never have sufficient cloud to generate rain, so we have reduced the threshold so that we do get cloud exceeding it and being converted to rain.